pycytominer/operations/transform.py

```diff
@@ -54,6 +55,11 @@ def __init__(self, epsilon=1e-6, center=True, method="ZCA"):
         ), f"Error {method} not supported. Select one of {avail_methods}"
         self.method = method
 
+        # PCA-cor and ZCA-cor require center=True
+        assert (
+            self.method not in ["PCA-cor", "ZCA-cor"] or self.center
+        ), "PCA-cor and ZCA-cor require center=True"
```

Comment:
```suggestion
        if self.method in ["PCA-cor", "ZCA-cor"]:
            assert(self.center), "PCA-cor and ZCA-cor require center=True"
```

This feels more readable to me

Reply:
Thanks. I've made these more readable and also changed them to exceptions